websites/S/Steam/presence.ts

```diff
@@ -3,335 +3,637 @@ const presence = new Presence({
+							presenceData.largeImageKey =
+								document.querySelector<HTMLMetaElement>(
+									"head > meta[property='og:image']"
+								).content;
+							presenceData.state = `Viewing ${document
+								.querySelector<HTMLTitleElement>("head > title")
+								.textContent.replace("Steam Community :: ", " ")}'s profile`;
+							presenceData.buttons = [
+								{
+									label: "Visit the profile page",
+									url: href,
+								},
+							];
+						}
+						break;
+					}
+					case "app": {
+						const title = document
+							.querySelector<HTMLTitleElement>("head > title")
+							.textContent.replace("Steam Community :: ", " ");
+						if (pathname.includes("/workshop/")) {
+							presenceData.details = "Steam Workshop";
+							presenceData.state = `Browsing through${title} submissions`;
+							presenceData.startTimestamp = browsingTimestamp;
+						} else if (pathname.includes("/discussions")) {
+							if (pathname.includes("/0/")) {
+								presenceData.details = "Viewing a discussion page";
+								presenceData.state = title
+									.replace("General ", " ")
+									.replace("::", "|");
+								presenceData.buttons = [
+									{
+										label: "Go to the discussion page",
+										url: href,
+									},
+								];
+							} else {
+								presenceData.state = document
+									.querySelector<HTMLTitleElement>("head > title")
+									.textContent.replace(" :: Steam Community", " ");
+								presenceData.startTimestamp = browsingTimestamp;
+							}
+						} else if (pathname.includes("/screenshots")) {
+							presenceData.state = `Screenshots for ${title}`;
+							presenceData.startTimestamp = browsingTimestamp;
+						} else if (pathname.includes("/images")) {
+							presenceData.state = `${title} Artworks`;
+							presenceData.startTimestamp = browsingTimestamp;
+						} else if (pathname.includes("/broadcasts")) {
+							if (pathname.includes("/watch/")) {
+								presenceData.startTimestamp = browsingTimestamp;
+								presenceData.state = "Watching a broadcast";
+								presenceData.buttons = [
+									{
+										label: "Watch now!",
+										url: href,
+									},
+								];
+							} else presenceData.state = `Browsing${title} live broadcasts`;
+						} else if (pathname.includes("/videos")) {
+							presenceData.state = `Browsing through${title} videos`;
+							presenceData.startTimestamp = browsingTimestamp;
+						} else if (pathname.includes("/allnews")) {
+							presenceData.state = `Reading news about${title}`;
+							presenceData.startTimestamp = browsingTimestamp;
+						} else if (pathname.includes("/guides")) {
+							presenceData.state = `Browsing in${title} guides`;
+							presenceData.startTimestamp = browsingTimestamp;
+						} else if (pathname.includes("/reviews")) {
+							presenceData.state = `Reading reviews about${title}`;
+							presenceData.startTimestamp = browsingTimestamp;
+						} else {
+							presenceData.details = `${title} Community Hub`;
```

Comment:
Maybe trim the title and add a space before the `${}` as its better readable that way

Reply:
Same as https://github.com/PreMiD/Presences/pull/7093#discussion_r1070241859
There's a space before it, so not a problem with having extra spaces here
honestly doesn't matter if use trim then add the space by ourselves...

Reply:
@EncryptedDev your opinion?